hosts/cassini/iss.py

```diff
@@ -415,27 +414,58 @@ def reset():
 
         Cassini.reset()
 
+
+
```

Comment:
Why was this changed from one blank line (correct) to three blank lines (incorrect). Please fix.